Automattic/newspack-blocks
# rename `article` by `post`

### All Submissions:






### Changes proposed in this Pull Request:

This PR renames the Homepage Articles to Homepage Posts, keeping the `article` key only for the block definition in order to don't break instances currently created.


### How to test the changes in this Pull Request:

1. Apply this brown changes
2. Restart your compiling process, since this PR rename the articles/posts block folder.
3. Tests with blocks already created before these changes.
4. Confirm that everything is working as expected. Pay attention to styles, too.
5. Create a new `Homepage posts` block.
6. Check that `article` has been replaced by `posts`: inserter, block preview, etc.

<img width="785" alt="Screen Shot 2019-11-25 at 11 43 31 AM" src="https://user-images.githubusercontent.com/77539/69549953-f3dc1380-0f78-11ea-83de-61612646cc25.png">


### Other information:
